PetApi/PriceRangeModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PetApi
+{
+    public class PriceRangeModel
+    {
+        public PriceRangeModel()
+        {
+        }
+
+        public PriceRangeModel(int minValue, int maxValue)
+        {
+            this.MinValue = minValue;
+            this.MaxValue = maxValue;
+        }
+
+        public int MinValue { get; set; }
+        public int MaxValue { get; set; }
+    }
+}
```

Comment:
maybe try query is better like url?max=xx&min=xx, because this model seems like a data class.